components/backup-stream/src/subscription_track.rs

```diff
@@ -479,15 +479,17 @@ impl TwoPhaseResolver {
         if !self.in_phase_one() {
             warn!("backup stream tracking lock as if in phase one"; "start_ts" => %start_ts, "key" => %utils::redact(&key))
         }
-        self.resolver.track_lock(start_ts, key, None)
+        // TODO: handle memory quota exceed.
+        let _ = self.resolver.track_lock(start_ts, key, None);
     }
 
     pub fn track_lock(&mut self, start_ts: TimeStamp, key: Vec<u8>) {
         if self.in_phase_one() {
             self.future_locks.push(FutureLock::Lock(key, start_ts));
             return;
         }
-        self.resolver.track_lock(start_ts, key, None)
+        // TODO: handle memory quota exceed.
```

Comment:
Will it impact correctness when track_lock return false here?  Because it actually does not track the lock effectively. If that, I think it's better to panic it for now other than silent failure.

Reply:
Yes, it does impact correctness.